src/archivist/index.js

```diff
@@ -130,7 +130,15 @@ export default class Archivist extends events.EventEmitter {
       }));
     } catch (error) {
       if (error instanceof FetchDocumentError) {
-        throw new InaccessibleContentError(error.message);
+        if (error.message.includes('EAI_AGAIN')) {
+          // DNS error is a network connectivity error or proxy related error. It does
+          // not necesarily mean the document is not accesible to all users but
+          // only to the Open Terms Archive engine at this specific time.
+          // As this is transient most of the time, consider it as a system error
```

Comment:
The problem with the sentence `It does not necesarily mean the document is not accesible to all users but only to the Open Terms Archive engine at this specific time.` is that it's true for some others errors (like 403, timed out, …) So I think we cannot use this as an argument to manage it this way.
I suggest :

```suggestion
          // EAI_AGAIN is a DNS lookup timed out error, which means it is a network connectivity error or proxy related error.
          // This operational error is mostly transient and should be handled by retrying the operation.
          // As there is no retry mechanism in the engine yet, crash the engine and leave it to the process manager to handle the retries and the delay between them.
```

Reply:
I agree your version is clearer, thanks